planemo/io.py

```diff
@@ -45,6 +45,26 @@ def communicate(cmds, **kwds):
     return output
 
 
+def copy(source, dest):
+    info("cp '%s' '%s'" % (source, dest))
+    shutil.copy(source, dest)
+
+
+def mkdir(path):
+    info("mkdir '%s'" % path)
+    os.mkdir(path)
+
+
+def makedirs(path):
+    info("mkdir -p '%s'" % path)
+    os.makedirs(path)
```

Comment:
A slight difference with between `mkdir -p` and `os.makedirs()` is that the former succeeds even if the directory already exists, while the latter fails. You may want to guard this with an `if not os.path.exists(path):`